flink-clients/src/test/java/org/apache/flink/client/program/rest/RestClusterClientTest.java

```diff
@@ -509,7 +536,7 @@ public void testListJobs() throws Exception {
 				Iterator<JobStatusMessage> jobDetailsIterator = jobDetails.iterator();
 				JobStatusMessage job1 = jobDetailsIterator.next();
 				JobStatusMessage job2 = jobDetailsIterator.next();
-				Assert.assertNotEquals("The job statues should not be equal.", job1.getJobState(), job2.getJobState());
+				Assert.assertNotEquals("The jobGraph statues should not be equal.", job1.getJobState(), job2.getJobState());
```

Comment:
previous message was fine ... also *status*

Reply:
Will fix it.